test/fixtures/scenarios/aws-landscape/0a055c90-4cb6-54ce-ac9c-ae3fedaf40d4_ha_cluster_discovery.json

```diff
@@ -222,7 +222,7 @@
                     {
                       "Id": "rsc_ip_HDD_HDB10-instance_attributes-ip",
                       "Name": "ip",
-                      "Value": "10.100.1.13"
+                      "Value": "10.100.1.1347"
```

Comment:
Is this a good IP? :thinking:

Reply:
oh boy, sorry i was testing stuff and I left this